tests/unit_test/api/api_processor_test.py

```
@@ -2820,7 +2820,7 @@ def test_validate_names_empty_fields(self):
     def test_validate_names_invalid_chars(self):
         values = {"first_name": "Invalid@Name", "last_name": "Last#Name"}
         with pytest.raises(ValueError,
-                           match="First name and last name can only contain letters, numbers, spaces and underscores."):
+                           match="First name and last name can only contain letters, numbers, spaces and underscores."):OthersOthers
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)